docker/node/entry.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+mkdir -p ~/RaiBlocks
+if [ ! -f ~/RaiBlocks/config.json ]; then
+  echo "Config File not found, adding default."
+  cp /usr/share/raiblocks/config.json ~/RaiBlocks/
```

Comment:
Isn't this the `node`'s existing default behavior?  Why reproduce it here?

Reply:
The default config here listens on IPv4, not IPv6, because Docker only works with the former.

Reply:
It also enables the RPC. You can view the `config.json` file in this PR for more details.

Reply:
It's a shame that it's another maintenance item but I see why it's needed now.